pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -6245,7 +6315,6 @@ func initGeoBuiltins() {
 		{"st_geogfromtext", "st_geographyfromtext"},
 		{"st_geomfromtext", "st_geometryfromtext"},
 		{"st_numinteriorring", "st_numinteriorrings"},
-		{"st_makepoint", "st_point"},
```

Comment:
we should keep the alias (i don't mind if st_point offers the Z/ZM variation), or define `st_point` again.

Reply:
I left `st_point` untouched. Previously, `st_makepoint` was aliased to use the `st_point` code.

Reply:
oh i see, read this the wrong way.